actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java

```diff
@@ -312,6 +324,38 @@ public Response getAllRecords(String keyspaceName, String tableName, RequestCont
     return response;
   }
 
+  @Override
+  public Response getAllRecords(
```

Comment:
is the existing getAllRecords() function calling this? Asking if this statements are duplicated in another function.

Reply:
yes, removed the duplicates, it is calling existing one getAllRecords(),